common/src/main/java/io/netty/util/collection/PrimitiveCollections.java

```diff
@@ -122,8 +166,7 @@ public boolean containsValue(Object value) {
      *
      * @param <V> the value type stored in the map.
      */
-    private static final class UnmodifiableIntObjectMap<V> implements IntObjectMap<V>,
-            Iterable<IntObjectMap.Entry<V>> {
+    private static final class UnmodifiableIntObjectMap<V> implements IntObjectMap<V> {
```

Comment:
Should `UnmodifiableIntObjectMap ` be "script templated" like the other classes?  Also the associated accessors in this class?